docs/conceptual/array.sum[^t]-function-[fsharp].md

```diff
@@ -36,15 +36,16 @@ The input array.
 
 
 ## Return Value
-
-The resulting sum.
+The sum of all elements of the array.
 
 
 ## Exceptions
-May throw an [OverflowException](https://msdn.microsoft.com/library/system.overflowexception.aspx), as the function uses the [checked (+) operator](https://msdn.microsoft.com/visualfsharpdocs/conceptual/checked.[-p-][%5Et1%2c%5Et2%2c%5Et3]-function-[fsharp]) for the additions.
+May throw an [OverflowException](https://msdn.microsoft.com/library/system.overflowexception.aspx) due to arithmetic overflows.
```

Comment:
I agree with the decision here to move the checked operator bit into the remarks.  The reason for throwing an `OverflowException` is due to arithmetic overflow, which should be pretty intuitive given that this is a `sum` function.  The fact that it uses the checked `+` operator is additional information, but isn't the reason why that exception may be thrown.